### ISSUE TITLE
`useRuntimeConfig is not defined`

### ISSUE THREAD

**Issue by rogadev** [nuxt-modules/kinde#62]
I directly cloned the template into my own repository. The only thing I've changed is removing the package lock and using pnpm to install the packages instead. Adding my own env variables to the .env file and attempting `pnpm run dev` renders the following issue:

```
[nuxt] [request error] [unhandled] [500] useRuntimeConfig is not defined
  at createSessionManager (C:\Development\maintenance\node_modules\.pnpm\@nuxtjs+kinde@0.1.7\node_modules\@nuxtjs\kinde\dist\runtime\server\middleware\kinde.mjs:17:18)   
  at Object.handler (C:\Development\maintenance\node_modules\.pnpm\@nuxtjs+kinde@0.1.7\node_modules\@nuxtjs\kinde\dist\runtime\server\middleware\kinde.mjs:5:32)
  at Object.handler (/C:/Development/maintenance/node_modules/.pnpm/h3@1.11.1/node_modules/h3/dist/index.mjs:1962:31)
  at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  at async Server.toNodeHandle (/C:/Development/maintenance/node_modules/.pnpm/h3@1.11.1/node_modules/h3/dist/index.mjs:2249:7)
[nuxt] [request error] [unhandled] [500] useRuntimeConfig is not defined
  at createSessionManager (C:\Development\maintenance\node_modules\.pnpm\@nuxtjs+kinde@0.1.7\node_modules\@nuxtjs\kinde\dist\runtime\server\middleware\kinde.mjs:17:18)   
  at Object.handler (C:\Development\maintenance\node_modules\.pnpm\@nuxtjs+kinde@0.1.7\node_modules\@nuxtjs\kinde\dist\runtime\server\middleware\kinde.mjs:5:32)
  at Object.handler (/C:/Development/maintenance/node_modules/.pnpm/h3@1.11.1/node_modules/h3/dist/index.mjs:1962:31)
  at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  at async toNodeHandle (/C:/Development/maintenance/node_modules/.pnpm/h3@1.11.1/node_modules/h3/dist/index.mjs:2249:7)
  at async ufetch (/C:/Development/maintenance/node_modules/.pnpm/unenv@1.9.0/node_modules/unenv/runtime/fetch/index.mjs:9:17)
  at errorhandler (C:\Development\maintenance\node_modules\.pnpm\nuxt@3.10.3_vite@5.1.5\node_modules\nuxt\dist\core\runtime\nitro\error.js:37:41)
  at async Server.toNodeHandle (/C:/Development/maintenance/node_modules/.pnpm/h3@1.11.1/node_modules/h3/dist/index.mjs:2256:9)
  ```
  
  This is directly from the nuxt-kinde template available on the Kinde + Nuxt starter on GitHub, linked to by the Kinde website.
  https://github.com/kinde-starter-kits/nuxt-starter-kit

**Comment by DanielRivers** [comment]
@rogadev This has been resolved already with https://github.com/nuxt-modules/kinde/pull/60.

Will be fixed in 0.1.8, in the mean time please use 0.1.6

**Comment by rogadev** [comment]
Sounds good, thanks @DanielRivers .

**Comment by DanielRivers** [comment]
@rogadev this has been released 0.1.8
